### DISCUSSION TITLE
Is there a way to configure wezterm to use my laptops touchscreen to scroll?

### DISCUSSION THREAD

**Discussion by anorman1967** [wezterm/wezterm#4327] [Q&A]
### What Operating System(s) are you running on?

Linux X11

### Which Wayland compositor or X11 Window manager(s) are you using?

Cinnamon 5.4.12

Operating System: Linux Mint 21
Kernel: Linux 5.15.0-84-generic
Architecture: x86-64

### WezTerm version

wezterm 20230712-072601-f4abf8fd

### Ask your question!

Is there a way to configure wezterm to use my laptops touchscreen to scroll? The only terminal I've ever found to do this out of the box is alacritty, and I would love to switch to wezterm as my daily driver, but I still use alacritty for development work for this one reason alone (being able to scroll code in neovim with a finger on the screen is a feature I have come to love).. 

Thank you

**Comment by wez** [comment]
Not currently. See:
* https://github.com/wez/wezterm/issues/792

**Reply by anorman1967** [reply]
Sorry, I didn't see that feature request.

That being said, just wanted to thank you again for the product.. Although I'll be using Alacritty for my development work, I am using wezterm for everything else, as I love it.. Thank you again!!!!

Andre
